docs(useEventListener): improve docs (#2105)
vueuse/vueuse: packages/core/useEventListener/index.md

  <div v-else ref="element">Div2</div>
</template>
```

> useEventListener will also return a function that you can call to remove the listener on demand.

Reviewer: ```suggestion
You can also call the returned to unregister the listener.
```

```ts
import { useEventListener } from '@vueuse/core'

const cleanup = useEventListener(document, 'keydown', (e) => { console.log(e.key) })

> cleanup(); // This will unregister the listener.

Reviewer: ```suggestion
cleanup() // This will unregister the listener.
```
